docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -571,6 +571,21 @@ the state of the table to a previous snapshot id::
 
     CALL iceberg.system.rollback_to_snapshot('testdb', 'customer_orders', 8954597067493422955)
 
+Register table
+----------------
+Iceberg supports creating table with existing metadata and data files.
+
+A SQL procedure ``system.register_table`` allows the caller to register a new table
+with existing metadata and data::
+
+    CALL iceberg.system.register_table('testdb', 'customer_orders', 'hdfs://hadoop-master:9000/user/hive/warehouse/customer_orders-581fad8517934af6be1857a903559d44')
```

Comment:
Use named parameters  in the example to provide a better explanation on how the procedure is being used.

Reply:
Addressed